toolchest_client/api/status.py

```diff
@@ -2,12 +2,29 @@
 toolchest_client.api.status
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-This module contains a Status class for status updates to the Toolchest API.
+This module contains a function to check pipeline_segment_instance statues and status enums.
```

Comment:
Typo, statuses

Reply:
Or was it?

<img width="246" alt="Screen Shot 2022-02-15 at 9 33 00 PM" src="https://user-images.githubusercontent.com/10551613/154191123-3821717c-d8a1-46af-af4e-fbdc2e0ac37b.png">

Reply:
Jk, will fix